apps/test-suite/tests/FirebaseJSSDK.js

```diff
@@ -191,6 +182,20 @@ export async function test({ describe, it, expect, beforeAll }) {
       });
     });
   });
+
+  describe('regression', () => {
+    describe('firebase/auth', () => {
+      // see: https://github.com/firebase/firebase-js-sdk/issues/5638
+      it('exports signInWithPhoneNumber', () => {
+        expect(signInWithPhoneNumber).not.toBe(undefined);
+      });
+
+      // see: https://github.com/firebase/firebase-js-sdk/issues/5638
+      it('exports PhoneAuthProvider', () => {
+        expect(PhoneAuthProvider).not.toBe(undefined);
+      });
+    });
+  });
```

Comment:
```suggestion
  describe('regression', () => {
    // see: https://github.com/firebase/firebase-js-sdk/issues/5638
    describe('firebase/auth', () => {
      it('exports signInWithPhoneNumber', () => {
        expect(signInWithPhoneNumber).not.toBe(undefined);
      });

      it('exports PhoneAuthProvider', () => {
        expect(PhoneAuthProvider).not.toBe(undefined);
      });
    });
  });
```
If it is the same link, we can mention it only once. 🙂

Reply:
Yeah, thats true. When writing this I thought more of "1 test per issue, every issue should have an explanation". But we can remove one of them :)